nvbench/detail/measure_cold.cuh

```diff
@@ -206,7 +211,10 @@ private:
       return;
     }
 
-    kernel_launch_timer timer(*this);
+    // disable use of blocking kernel for warm-up run
+    // see https://github.com/NVIDIA/nvbench/issues/240
+    constexpr bool disable_blocking_kernel = true;
+    kernel_launch_timer timer(*this, disable_blocking_kernel);
```

Comment:
Please, file an issue to investigate if we can safely remove code below after this change:
```
  set_env("CUDA_MODULE_LOADING", "EAGER");
```